v2/gcs-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dao/BaseDao.java

```diff
@@ -110,7 +106,7 @@ public void batchWrite(List<String> batchStatements) throws SQLException {
                   + ", will retry : "
                   + e.getMessage());
           // gives indication that the shard is being retried
-          Metrics.counter(MySqlDao.class, "mySQL_retry_" + shardId).inc();
+          Metrics.counter(BaseDao.class, "BaseDao_retry_" + shardId).inc();
```

Comment:
could you rename to Db_retry

Reply:
Renamed.

Reply:
The metric will be visible to customer, so they will not understand what BaseDao retry means. So db_retry or db_connection_retry would help.

Reply:
Had renamed this metric to DB_retry_

Reply:
All metrics have been renamed to `db_connection_retry`